sensor/common/sensor/sensor.go

```diff
@@ -97,6 +101,12 @@ func (s *Sensor) AddAPIServices(services ...pkgGRPC.APIService) {
 	s.apiServices = append(s.apiServices, services...)
 }
 
+// AddNotifiable adds a common.Notifiable component to the list of components that will be notified of any connectivity
```

Comment:
is it *only* connectivity changes?

Reply:
As of now, yes. But we can't predict if this will be used in the future for other things. I'd prefer be more explicit about its use current use in the comment.